src/delta_lake/delta_service.py

```diff
@@ -115,6 +118,74 @@ def _check_exists(database: str, table: str) -> bool:
     return True
 
 
+def _generate_cache_key(prefix: str, params: Dict[str, Any]) -> str:
```

Comment:
```suggestion
def _generate_cache_key(namespace: str, params: Dict[str, Any]) -> str:
```
I think that's what the prefix is in this case - separating json blobs that have the same contents but different semantics

Reply:
no. It has nothing to do with namespace - each endpoint method should have their own prefix, e.g. sample, count, query, etc.

Reply:
That's a namespace to me, but I'm not religious about this